src/Compilers/CSharp/Portable/Binder/PatternSwitchBinder2.cs

```diff
@@ -48,24 +49,49 @@ internal override BoundStatement BindSwitchExpressionAndSections(SwitchStatement
             var boundSwitchExpression = SwitchGoverningExpression;
             diagnostics.AddRange(SwitchGoverningDiagnostics);
 
-            BoundPatternSwitchLabel defaultLabel;
-            bool isComplete;
-            ImmutableArray<BoundPatternSwitchSection> switchSections =
-                BindPatternSwitchSections(originalBinder, out defaultLabel, out isComplete, out var someCaseMatches, diagnostics);
+            ImmutableArray<BoundPatternSwitchSection> switchSections = BindPatternSwitchSections(originalBinder, diagnostics, out BoundPatternSwitchLabel defaultLabel);
             var locals = GetDeclaredLocalsForScope(node);
             var functions = GetDeclaredLocalFunctionsForScope(node);
-            BoundDecisionDag decisionDag = null; // we'll compute it later
-            return new BoundPatternSwitchStatement(
+            BoundDecisionDag decisionDag = new DecisionDagBuilder(this.Compilation).CreateDecisionDag(
+                syntax: node,
+                switchExpression: boundSwitchExpression,
+                switchSections: switchSections,
+                defaultLabel: defaultLabel?.Label ?? BreakLabel);
```

Comment:
>BreakLabel [](start = 53, length = 10)

It is not obvious why it is the right thing to do to pass ```BreakLabel``` as default label.

Reply:
Because the default action in a switch that has no `default` label is to break out of (i.e. jump to the end of) the switch.

Reply:
>Because the default action in a switch that has no default label is to break out of (i.e. jump to the end of) the switch.

Consider adding a comment, or naming the parameter differently.

Reply:
Addressed in https://github.com/dotnet/roslyn/pull/24059/commits/114a56e9077ed03f08443889b69cae6972a4acc6 of https://github.com/dotnet/roslyn/pull/24059